script/Ciel/Day 5/05_01_CIEL05_4.txt

```diff
@@ -891,17 +893,19 @@ To escape that cold, he abandoned his own knighthood.
 -- Page 725, Offset 12814.
 -- <拐|かどわ>かされたとはいえ。
 // Is this literal? If there's no lore point on this it feels more figurative to me (hostage to his desires)
-Even though he had been taken prisoner―――
+// this doesn't mean 'being abducted', it means 'being deceived', it's a very alternative and weird definition of this word, however, Nasu absolutely loves it. You can see him use it in the same sense later on in the game, Offset 37301, Offset 39009, Offset 41118, all in the sense of Roa tricking Arcueid into drinking his blood. Though looking at the script, they have all been mistranslated there as well.
+//Oh, and the person who deceived him was Roa, hence his hatred of him
+And though he may have been deceived―――
 }
 [sha:759c237218199e7b8b6aa4657904bcdeaa2eb59a]{
 -- Page 725, Offset 12815.
 -- 唯一の陽差しを、自らの手で閉ざしたのだ。
-He blocked off his only ray of sunshine with his own hands.
+He still shut it out by his own two hands; his sole sunshine.
```

Comment:
Well, obviously I disagree, but I will ask you to at least include 'shut out' instead of 'blocked off', because the later lacks the emphatic quality of showing that he killed his beloved Zaria.

Reply:
Those kinda seem identical to me, but if that's the context I'd say 'cut off', or 'snuffed out'

Reply:
It merely comes down to the emphatic quality.
Shut out sounds more dramatic, more final, while block off sounds like someone putting a hand in front of their face because it was bothering them.

Reply:
Eh I think that's heavily dependent on what each person imagines for each. To me shut out is the weaker of the two. Regardless, I think snuff is probably unambiguous?

Reply:
The difference is that Bruce Springsteen made 'Shut out the light' be about someone afraid of the light leaving them in the darkness, while 'Block off the light' isn't a beloved pop-culture classic that exists!!

Reply:
Original phrasing with 'cut off' is best for me.

Reply:
"snuff out" makes the most sense to me but I'm fine with _either_ of the current interpretations but willing to err on the side of the Petrikow Line if you really think it makes a difference lore-wise